en/flight_controller/raspberry_pi_pilotpi.md

```diff
@@ -0,0 +1,177 @@
+# RPi PilotPi Shield
+
+> **Warning** PX4 does not manufacture this (or any) autopilot.
+  Contact the [manufacturer](mailto:lhf2613@gmail.com) for hardware support or compliance issues.
+
+<span></span>
+> **Warning** PX4 support for this flight controller is [experimental](../flight_controller/autopilot_experimental.md).
+
+The *PilotPi* shield is a fully functional solution to run PX4 autopilot directly on Raspberry Pi.
+It is designed to be a low-cost but highly scalability platform with continuous updates from both Linux and PX4 sides.
+No proprietary driver is required, as all components have upstream support from RPi and PX4 community.
+PCB and schematic are open source as well.
```

Comment:
Link to where these open source files are?

Reply:
I haven't decide where to store those resources. It's now available at https://github.com/Airunix-2020-SCU/AirPi2_PCB , but this may not be the final location.

Reply:
OK. Remember to add those links at some point when you have a home for them.